interactive_engine/src/instance-manager/src/main/java/com/alibaba/maxgraph/admin/controller/InstanceManagerController.java

```diff
@@ -195,9 +200,9 @@ public CreateInstanceEntity createInstance(@RequestParam("graphName") String gra
         int frontendPort = 0;
 
         String schemaPath = "/tmp/" + graphName + ".json";
-        if (StringUtils.isEmpty(SecurityUtil.pathFilter(schemaPath))) {
+        if (!isValidGraphName(graphName) || StringUtils.isEmpty(SecurityUtil.pathFilter(schemaPath))) {
             createInstanceEntity.setErrorCode(-1);
-            createInstanceEntity.setErrorMessage("Invalid graph name=>" + graphName);
+            createInstanceEntity.setErrorMessage("Invalid graph name");
```

Comment:
Apparently there would be two reasons, one is the `graphName` is not valid, one is the `schemaPath` is empty. we should be able to distinguish them in the error message. So maybe put these validations in separate branches would be more clear.